libs/dh/globalization/assets-localization/src/assets/i18n/en.json

```
@@ -16,7 +16,81 @@
     "overview": {
       "breadcrumb": {
         "meteringPoints": "Metering points"
+      },
+      "meteringPointType": "Type of Metering Point",
+      "meteringMethod": "Metering Method",
+      "gsrnNumber": "Metering Point ID",
+      "connectionState": "Connection State",
+      "settlementMethod": "Settlement Method",
+      "readingOccurrence": "Meter Reading Periodicity",
+      "meteringPointNotFound": {
+        "title": "Vi fandt desværre intet målepunkt [EN]",
```

To match the translation used by search:
```suggestion
        "title": "Sorry, we did not find a metering point match",
```